### ISSUE TITLE
Add @@ modifier to iterate over all maps, sections, ...

### ISSUE THREAD

**Issue by radare** [radare/radare2#8980]
changing blocksize and such

**Comment by radare** [comment]
We have to define the syntax for this operator. we need to specify the permissions ? only maps with 'x'? for sections, that only makes sense in PA mode.

**Comment by radare** [comment]
@@iS implemented here: 6cdd273d5c5bec51f7ffee5b09f98e8b69f25149

**Comment by Maijin** [comment]
isn't that @@@ ?

**Comment by radare** [comment]
Right... we should improve the help messages and probably clarify that @@@ changes blocksize and @@ only offset. And review all those suffix operators.

**Comment by ret2libc** [comment]
Something like `@@@o:rwx` could work to iterate over all maps?

**Comment by radare** [comment]
the problem is not iterating maps. but to specify offset+size on each element.

maybe we can define a syntax like ! and reuse other commands output like this

x @@c:omq

(and make omq output be less quiet and show vaddr!size of each map)

**Comment by ret2libc** [comment]
>
> the problem is not iterating maps. but to specify offset+size on each
> element.
>
> What do you mean to specify offset+size? Why do you need to specify them?
Once you do `@@@o`, radare2 should be able to retrieve all the info about
maps, right?

I think I don't understand the issue.

**Comment by XVilka** [comment]
Then also should be documented in radare2book

**Comment by radare** [comment]
What about discussing the proper way instead of documenting and extending what we know already its dupped?
